### PR TITLE
Automatically convert image and html like assignments to uploads

### PR DESCRIPTION
**Pull request by shnela**

### CODE REVIEW

File: src/neptune/new/types/type_casting.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+)
+from neptune.new.types.atoms.datetime import Datetime
+from neptune.new.types.atoms.float import Float
+from neptune.new.types.atoms.string import String
+from neptune.new.types.namespace import Namespace
+from neptune.new.types.value import Value
+from neptune.new.types.value_copy import ValueCopy
+
+
+def cast_value(value: Any) -> Value:
```
Comment (Raalsky) [inline src/neptune/new/types/type_casting.py:44]: As we extracted this func (which is so cool!) what do you think about using a single-dispatch? https://docs.python.org/3.7/library/functools.html#functools.singledispatch You should be able to split this function into multiple one that will be triggered base on arguments type.

File: src/neptune/new/metadata_containers/metadata_container.py [hunk 2]
```diff
@@ -14,14 +14,12 @@
 # limitations under the License.
```
Comment (Raalsky) [inline src/neptune/new/metadata_containers/metadata_container.py:14]: CHANGELOG?

File: src/neptune/new/types/type_casting.py (hunk 1 above)
Comment (shnela) [reply src/neptune/new/types/type_casting.py:44]: I think it's a good idea.
Let's try it.

Comment (shnela) [reply src/neptune/new/types/type_casting.py:44]: Or maybe not.
We do casting not only based on type, but also on predicate which looks like it's not supported by singledispatch.

**Review by Raalsky — APPROVED** [review]

**Pull request merged**